### DISCUSSION TITLE
How can I override registry key for Ubisoft and Far Cry 3

### DISCUSSION THREAD

**Discussion by matyugovich** [moraroy/NonSteamLaunchers-On-Steam-Deck#423] [Q&A]
Sorry for being such a noob. A im new to Linux gaming. The is a registry key in /home/matyugovich/.local/share/Steam/steamapps/compatdata/UplayLauncher/pfx/user.reg
[Software\\Ubisoft\\Far Cry 3] 1725976797
#time=1db0389b871954a
"ExecutableName"="bin/FC3Updater.exe"
"StartArguments"=""

How can i override it? I want to change FC3Updater.exe to farcry3_d3d11.exe. Changing it in this file doest nothing

**Comment by moraroy** [comment]
@matyugovich I have no idea as I've never done that before!
